lib/synchronizer.php

```diff
@@ -54,10 +60,21 @@ public function __construct($dirname, array $files)
      */
     abstract protected function editItem(rex_developer_synchronizer_item $item);
 
+    /**
+     * The method is called, when an existing item is deleted by the file system (`FORCE_FILES` is activated)
+     *
+     * Use the method to delete the item in the base system
```

Comment:
Base system

Reply:
Kommt per copy/paste von den anderen Methoden.

Das Ding ist, das developer-Addon ist nicht nur darauf ausgelegt, in die DB zu synchronisieren, sondern theoretisch sonst wo hin. Also die eine Seite sind immer Dateien (Developer-Dateien). Die andere Seite bestimmen die Synchronizer. Die drei default Synchronizer (Templates, Module, Actions) syncen halt in die DB. Diese Basis-Synchronizer-Klasse lässt das aber offen.